### ISSUE TITLE
Incorrect version number for Gurobi

### ISSUE THREAD

**Issue by GustavBjordal** [MiniZinc/libminizinc#411]
The MiniZinc IDE (and `minizinc`) lists gurobi as version 7.5.1 for me, but I am running 9.0.3.

If I run gurobi via the CLI with `-v` I get the following output:

```
MIP solver plugin, compiled Oct  6 2020, using: MIP wrapper for Gurobi library 7.5.1.  Compiled  Oct  6 2020  03:25:01
Gurobi Optimizer version 9.0.3 build v9.0.3rc0 (linux64)
```
So it seems that 9.0.3 is running after all. 
Is this a bug or is there some semantics of the version number listed by the IDE that I am misunderstanding?

**Comment by cyderize** [comment]
I think this is probably due to the way we handle the DLL flags.

I suspect what's happening is that MiniZinc is finding Gurobi 7.5.1 in `/opt/gurobi751` (one of the built in search paths) and that's the version number that it prints. This version number gets set before handling the `--gurobi-dll` flag, which is why it's wrong (although solving will use the version from the `--gurobi-dll` flag as you have seen).

I'm working on a fix for this at the moment.

**Comment by cyderize** [comment]
This is fixed now in 2.5.1
